Editor/AGS.Types/EditorFeatures/MenuCommand.cs

```diff
@@ -14,9 +14,9 @@ public static MenuCommand Separator
 			get { return new MenuCommand(null, MenuCommand.MENU_TEXT_SEPARATOR); }
 		}
 
-		public MenuCommand(string itemName, IList<MenuCommand> subCommands)
+		public MenuCommand(string itemName, IList<MenuCommand> subCommands, string itemId = null)

```

Comment:
I have a proposal here, to instead make 2 separate constructors, one of which has itemId as a first parameter, and another does not have it.

My feeling is that this will be more consistent with existing constructors which have itemId first param, as it's the priority property for the MenuCommand, and more obvious for coders.
(Note that since this is in AGS.Type, it's practically a Editor's api)

Reply:
I can't actually do this because a different constructor gets used if I move the `IList<MenuCommand>` to the right, the null for iconKey or shortcutKey gets passed to subCommands. I need to think how to do this.

Reply:
I did it like this https://github.com/adventuregamestudio/ags/pull/2465/commits/20f7833f719beb8e97c181b50823ca4e3089c153